WireAnalytics/Sources/WireAnalytics/Events/Segmentation/Predefined/Segmentation+Conversation.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+public extension AnalyticsEvent.Segmentation {
+
+    enum Conversation {
+
+        public enum ConversationType: String {
+            case group
+            case oneOnOne = "one_to_one"
+        }
+
+        // https://wearezeta.atlassian.net/browse/WPB-12199?focusedCommentId=132080
+        @available(*, deprecated, message: "Use `ConversationType`.")
+        enum LegacyConversationType: String {
+            case group
+            case oneOnOne = "one_on_one"
+            case unknown
+        }
```

Comment:
Is this being used still?

Reply:
Yes, for the (old) `"group_type"` segmentation.

Reply:
> How about deprecating the `unknown` enum case?

The raw value for `oneOnOne` is also different: `"one_on_one"` vs `"one_to_one"`